### PR TITLE
fix(dedicated): cpanel license upgrade

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #DTRSD-17392
| License          | BSD 3-Clause

## Description

Add upgrade option for cpanel license. Seems like this option alone is broken for license upgrade

### :flags: Translations

- [ ] _to be defined_

### CODE REVIEW

**Review by antleblanc — APPROVED** [review]
@ganeshkumar1989 Changes looks good to me.
Could you please rebase on `master` branch? In order to be able to deploy it.

Thank you!

File: packages/manager/apps/dedicated/client/app/license/upgrade/CPANEL.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<div class="control-group">
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/license/upgrade/CPANEL.html:1]: file was deleted ?

Comment (ganeshkumar1989) [reply packages/manager/apps/dedicated/client/app/license/upgrade/CPANEL.html:1]: Not sure about it...

**Comment by ganeshkumar1989** [comment]
> @ganeshkumar1989 Changes looks good to me.
> Could you please rebase on `master` branch? In order to be able to deploy it.
> 
> Thank you!

This is done. Thanks :)

**Review by antleblanc — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report translator-pull#614.0 ✘  
* 
  * pull ✘

**Review by JeremyDec — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report test#16103.0 ✘  
* Stage 1
  * lint ✘  
  * SemVer ✔

**Comment by antleblanc** [comment]
Closing this Pull Request.

cc @radireddy

**Pull request closed without merging**